packages/docs/docs/Usage/theming.md

```diff
@@ -1,11 +1,12 @@
-# Embedded Chat: A Staple in Excellent Customer Service
+---
+title: Theming Guide
+---
+# Theming
 
 An easy-to-use, full-stack component (React.js + backend behaviors) for embedding Rocket.Chat into your web app.
```

Comment:
Remove this !

Reply:
The title themeing guide is for the the title in sidebar

Reply:
I mean remove this : An easy-to-use, full-stack component (React.js + backend behaviors) for embedding Rocket.Chat into your web app from here